en/assembly/quick_start_pixhack_v3x.md

```diff
@@ -0,0 +1,61 @@
+#Pixhack V2/V3/V3X wiring quick start
+
+The quick start guide shows how to power the Pixhack V2/V3 flight controller and connect its most important peripherals.
+
+
+![Pixhack v3x](../../assets/flight_controller/pixhack_v3x/pixhack_v3x.png)
+
+###Mount and Orient Controller
+
+Pixhack V2/V3/V3X should be mounted on the frame using vibration-damping foam pads (included in the kit). It should be positioned as close to your vehicle’s center of gravity as possible, oriented top-side up with the arrow pointing towards the front of the vehicle.
+
+<img src="../../assets/flight_controller/pixhack_v3x/pixhack_v3_vehicle_feont.jpg" align="center"/>
```

Comment:
File name appears to have typo error: _feont.jpg (ie you probably mean "front"). It does match file.